osrdyne/src/drivers/docker.rs

```diff
@@ -47,11 +50,17 @@ impl DockerDriver {
         amqp_uri: String,
         worker_pool: String,
     ) -> DockerDriver {
+        let version_identifier = std::env::var("OSRD_GIT_DESCRIBE")
+            .unwrap_or_else(|_| format!("run-{}", Uuid::new_v4()));
+
+        let hashed = format!("{:x}", Sha256::digest(version_identifier))[..16].to_string();
```

Comment:
Why only the first 16 bytes?

Reply:
Because there is a capacity limit on labels on k8s.